modules/market/src/main/java/com/opengamma/strata/market/param/TenorDateParameterMetadata.java

```diff
@@ -62,7 +62,6 @@
    * @return the parameter metadata based on the tenor
    */
   public static TenorDateParameterMetadata of(LocalDate date, Tenor tenor) {
-    ArgChecker.notNull(date, "date");
```

Comment:
I think this should be left in - when one arg checker is needed, I would usually check everything